src/rgw/rgw_lc_s3.cc

```diff
@@ -102,6 +109,51 @@ bool LCFilter_S3::xml_end(const char* el) {
   return !(single_cond && num_conditions > 1);
 }
 
+bool LCTransition_S3::xml_end(const char* el) {
```

Comment:
We shouldn't use the old style xml decoding api. See use of `RGWXMLDecoder::decode_xml()`, e.g., at `rgw_xml_enc.cc`

Reply:
done